less/fontello-icon-fonts.less

```diff
@@ -15,7 +15,7 @@
 @media screen and (-webkit-min-device-pixel-ratio:0) {
   @font-face {
     font-family: 'fontello-icon-fonts';
-    src: url('../font/fontello-icon-fonts.svg?7699025#fontello-icon-fonts') format('svg');
+    src: url('../../font/fontello-icon-fonts.svg?7699025#fontello-icon-fonts') format('svg');
   }
```

Comment:
I suppose it doesn't hurt - we're not going to be using this repo for anything other than Ace